### ISSUE TITLE
Tax Amount wrong on checkout

### ISSUE THREAD

**Issue by krebbi** [concrete5-community-store/community_store#49]
when Tax Settings is set to "Yes, I will enter product prices INCLUSIVE of tax" the Tax is wrong on checkout. 

![bildschirmfoto 2016-04-07 um 15 05 53](https://cloud.githubusercontent.com/assets/3884521/14352268/efbc07e4-fcd3-11e5-91d0-b539730c4652.png)

19% included in 399,- should be 63,74 and not 33,53

**Comment by Mesuva** [comment]
Yes, this certainly was wrong.
I've made a fix via: https://github.com/concrete5-community-store/community_store/commit/4a3e7053cfa6f0c2abb72a117a184256f798c076

The issue was that we have a simple 10% GST here in Australia, which means we can do a quick 'divide by 11' shorthand to get our tax value. But I'd made a silly assumption as to the maths behind how that worked.

Let me know how you go with the update and I'll close this issue when you're happy it has been resolved.
